desc/objectives/_geometry.py

```diff
@@ -238,6 +240,8 @@ def __init__(
     ):
         if target is None and bounds is None:
             target = 1
+        if grid is not None:
+            errorif_sym(grid, name)
```

Comment:
do we need to check this for other objectives?

Reply:
aspect ratio is fine as is. I checked other objectives were also fine. (haven't looked at new coil objectives)